bigquery/google/cloud/bigquery/table.py

```diff
@@ -50,6 +50,8 @@
     '{:d} bytes remaining.')
 _DEFAULT_NUM_RETRIES = 6
 
+_FakeResponse = collections.namedtuple('_FakeResponse', ['status'])
```

Comment:
I have a more comprehensive plan for this. I didn't meant to commit this (yet). :)